specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/networkWatcher.json

```diff
@@ -3480,19 +3480,16 @@
         "roundTripTimeMin": {
           "description": "Minimum roundtrip time in milliseconds.",
           "readOnly": true,
-          "format": "int32",
           "type": "integer"
         },
         "roundTripTimeAvg": {
           "description": "Average roundtrip time in milliseconds.",
           "readOnly": true,
-          "format": "int32",
```

Comment:
why do we remove this format?

Reply:
talked offline. will add this format back next week since @junyezhu need confirm with one team member who is on vacation right now.